packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -273,6 +273,16 @@
         }
     ],
     "metrics": [
+        {
+            "name": "aws_fileSystem",
```

Comment:
in https://github.com/aws/aws-toolkit-common/pull/793 I proposed a new `ide_` prefix. maybe it fits better? filesystem operations are not really related to "aws"

Reply:
This metric is temporary as I'm not 100% sure this PR fixes the underlying issue and we need something quick.
I'll update to `ide_`, but we should probably discuss a better overall metric like the one you suggested